src/nginx/varify/varify.go

```diff
@@ -43,6 +51,40 @@ func main() {
 			}
 			return fmt.Sprintf("load_module %s.so;", filepath.Join(pathToModules, name))
 		},
+		"nameservers": func() string {
+			resolvConfFile, err := os.Open(resolvConfPath)
+			if err != nil {
+				log.Printf("Could not open %s file for reading. "+
+					"The default nameservers %s will be used. Error: %s", resolvConfPath, defaultNameserver, err)
+				return defaultNameserver
+			}
+			defer resolvConfFile.Close()
+			scanner := bufio.NewScanner(resolvConfFile)
+			var nameServers strings.Builder
+			for scanner.Scan() {
+				var line = scanner.Text()
+				// Lines that contain a semicolon (;) or hash character (#) in the first column are treated as comments.
+				if len(line)>0 && line[0]!='#' && line[0]!=';' {
+					line = strings.TrimSpace(line)
+					matches, _ := regexp.MatchString(`nameserver \d+\.\d+\.\d+\.\d+`, line)
```

Comment:
Specific reason to parse manually? How about https://golang.org/pkg/net/?m=all#dnsConfig ?

Reply:
Sounds OK. But I can't find a way to use this functions... I think it's for internal only use? 
`dnsConfig` goes to `systemConf` and it's used only by `lookup.go`?
I don't have experience in Go.